src/mplhep/plot.py

```diff
@@ -482,6 +550,32 @@ def hist2dplot(
     H = hist.values()
     xbins, xtick_labels = get_plottable_protocol_bins(hist.axes[0])
     ybins, ytick_labels = get_plottable_protocol_bins(hist.axes[1])
+
+    # Show under/overflow bins
+    # "show": Add additional bin with 5 times bin width
+    if flow == "show":
```

Comment:
I am a bit worried this will end up with weird proportions where the informative part of the hist is just a small thing in the centre with huge borders

Reply:
or we can move to 2 times bin width?